cluster-provision/centos8/Dockerfile

```diff
@@ -1,18 +1,17 @@
 
-#FROM fedora:31
-FROM fedora@sha256:8fa60b88e2a7eac8460b9c0104b877f1aa0cea7fbc03c701b7e545dacccfb433
+FROM quay.io/fedora/fedora@sha256:38813cf0913241b7f13c7057e122f7c3cfa2e7c427dca3194f933d94612e280b
 
 ARG centos_version
 
-RUN dnf -y update nettle && dnf -y install jq iptables iproute dnsmasq qemu openssh-clients screen && dnf clean all
+RUN dnf -y install jq iptables iproute dnsmasq qemu openssh-clients screen && dnf clean all
```

Comment:
Can you explain why nettle is not required ? 
(in PR desc / commit preferably if possible please)

Reply:
Yes - when I was running this - the dnf update was not actually updating anything - nothing to update.